### PR TITLE
New package: DimisAIO.DindeGDPS version 2024.01.26.0

### PR DESCRIPTION
**Pull request by masckmaster2007**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136706)

### CODE REVIEW

**Comment by masckmaster2007** [comment]
@microsoft-github-policy-service agree

File: manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DimisAIO.DindeGDPS
+PackageVersion: 2024.01.26.0
+PackageLocale: en-US
+Publisher: DimisAIO
+PackageName: DindeGDPS
```
Comment (stephengillie) [inline manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml:8]:
```suggestion
PackageName: DindeGDPS
PackageUrl: https://dl.dindegmdps.us.to/
```

**Comment by masckmaster2007** [comment]
Do I have to do something?
Did I mess up?

**Comment by masckmaster2007** [comment]
@stephengillie do you think there's anything else I should check out for?

File: manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DimisAIO.DindeGDPS
+PackageVersion: 2024.01.26.0
+PackageLocale: en-US
+Publisher: DimisAIO
+PackageName: DindeGDPS
+License: None
+Copyright: Copyright (C) 2024 DimisAIO
+ShortDescription: DindeGDPS Installer
```
Comment (Trenly) [inline manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml:11]: Please provide a better description of what this package is, does, or is used for

File: manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DimisAIO.DindeGDPS
+PackageVersion: 2024.01.26.0
+PackageLocale: en-US
+Publisher: DimisAIO
+PackageName: DindeGDPS
+License: None
```
Comment (Trenly) [inline manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by masckmaster2007 — COMMENTED** [review]
sorry for the late reply
feel free to tell me if I have anything else to change!

File: manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml (hunk 2 above)
Comment (masckmaster2007) [reply manifests/d/DimisAIO/DindeGDPS/2024.01.26.0/DimisAIO.DindeGDPS.locale.en-US.yaml]: I hope I updated it right, never did this before...

**Review by stephengillie — APPROVED** [review]

**Pull request merged**